dojo/tools/jfrog_xray_on_demand_binary_scan/parser.py

```diff
@@ -0,0 +1,224 @@
+                severity_desc += item["name"] + "\n" if item.get("name") else ""
+                severity_desc += item["description"] + "\n" if item.get("description") else ""
+    return severity_desc, remediation
+
+
+def get_component(vulnerability):
+    mitigation = ""
+    gav = ""
+    impact = "**Impact paths**\n"
+    if "components" in vulnerability:
+        components = vulnerability["components"]
+        gav = next(iter(components))
+        component = components[gav]
+        fixed_versions = component.get("fixed_versions")
+        if fixed_versions:
+            mitigation = "**Versions containing a fix:**\n"
+            mitigation = mitigation + "\n".join(fixed_versions)
+        if "impact_paths" in component:
+            impact_paths = component["impact_paths"][0]
+            for item in impact_paths:
+                if "component_id" in item:
+                    component_id = item["component_id"]
+                    impact = impact + "\n" + component_id
+                if "full_path" in item:
+                    full_path = item["full_path"]
+                    impact = impact + "\n" + full_path
+        return gav, mitigation, impact
+
+
+def get_version_vulnerability(vulnerability):
+    if "vulnerable_versions" in vulnerability["component_versions"]:
+        extra_desc = "\n**Versions that are vulnerable:**\n"
+        extra_desc += "\n".join(vulnerability["component_versions"]["vulnerable_versions"])
+        return extra_desc
+    return "None"
+
+
+def get_provider(vulnerabiity):
+    if "component_versions" in vulnerabiity:
+        provider = vulnerabiity.get("component_versions").get("more_details").get("provider")
+        if provider:
+            provider += f"\n**Provider:** {provider}"
+            return provider
+    return ""
+
+
+def get_ext(vulnerability):
+    if "EXT" in vulnerability:
+        return vulnerability["EXT"]
+    return ""
+
+
+def get_cve(vulnerability):
+    if "cves" in vulnerability:
+        cves = vulnerability["cves"]
+        return cves
+    return []
+
+
+def get_item(vulnerability, test):
+    severity_justification, remediation = get_severity_justification(vulnerability)
+    severity = get_servery(vulnerability)
+    references = get_references(vulnerability)
+    vulnerability_ids = list()
+    cwe = None
+    cvssv3 = None
+    cvss_v3 = "No CVSS v3 score."
+    extra_desc = ""
+    # Some entries have no CVE entries, despite they exist. Example CVE-2017-1000502.
+    cves = get_cve(vulnerability)
+    if len(cves) > 0:
+        for item in cves:
+            if item.get("cve"):
+                vulnerability_ids.append(item.get("cve"))
+        # take only the first one for now, limitation of DD model.
+        if len(cves[0].get("cwe", [])) > 0:
+            cwe = decode_cwe_number(cves[0].get("cwe", [])[0])
+        if "cvss_v3" in cves[0]:
+            cvss_v3 = cves[0]["cvss_v3"]
```

Comment:
Not sure but I suppose this is the correct form. Can you double-check it by unittests, please?
```suggestion
        if "cvss_v3_vector" in cves[0]:
            cvss_v3 = cves[0]["cvss_v3_vector"]
```